app/story/story_controller.py

```diff
@@ -206,10 +106,6 @@ def validate_references():
                "tasks": set(),
                }
 
-    story = None
-    with open(Config.STORY_FILE, "r") as story_file:
-        story = json.loads(story_file.read())
-
     existing_story_points = story["story_points"].keys()
```

Comment:
you don't use the impored `story_points` variable here.
also, I think, it does not make sense to add all existing story points to the referenced story points (see `# Actual story points`), the referenced story points should only be derived from the start point and the paths. I suggest the following:
```python
    existing_story_points = story_points.keys()
    referenced_story_points = []
    # Start
    referenced_story_points.append(story["start_point"])
    # Path Destinations
    for story_point in story_points.values():
        destinations = story_point["paths"].values()
        referenced_story_points.extend(destinations)
```